features/collections/collection_create.feature

```diff
@@ -153,7 +154,8 @@ Given I have the collection "Scotts Collection" with name "scotts_collection"
     And I fill in "Collection name" with "child_collection"
     And I press "Submit"
   Then I should see "Collection was successfully created"
-  When I go to the collections page
+  When all indexing jobs have been run 
```

Comment:
This can be removed now that `I go to the collections page` runs the indexing jobs.